### ISSUE TITLE
XSS vulnerability

### ISSUE THREAD

**Issue by callmewind** [groveco/django-sql-explorer#267]
Hi, first of all, thanks for this project; it saved me a lot of time.
I've just discovered that ouput from database is not sanitized berfore rendering it in html, and as far as I know, this makes this application vulnerable to XSS attacks.
It would be nice that EXPLORER_TRANSFORMS could skip the html sanitization, but documentation should clarify that it's use with char fields can be dangerous.

**Comment by JshWright** [comment]
This appears to be an intentional choice (autoescaping is turned off in the two templates that handle rendering of data). I would be curious what the reasoning is for this is... It's a very unsafe default, and certainly violates the "principle of least surprise" mentioned in the README.

**Comment by chrisclark** [comment]
I supposed this is true...the idea is that you can write sql like:

```
SELECT '<a href=http://site.com/user/' || id || '>' || profile_pic || '</a>' FROM users
```

And, for instance, render a gallery of users' profile photos in the results area. But I do see the issue -- it just doesn't come up with the type of data we store in our main explorer target.

I think the fix you proposed @JshWright -- #286 -- makes a lot of sense.
